deploy/all-in-one.yaml

```diff
@@ -1138,7 +1138,7 @@ spec:
     spec:
       containers:
       - args:
-        - --atlas-domain=https://cloud-qa.mongodb.com
+        - --atlas-domain=https://cloud.mongodb.com
```

Comment:
I think this made its way into the clusterwide-config.yaml too

Reply:
Yes, I fixed 2 more files